PRIVACY.md

```diff
@@ -0,0 +1,48 @@
+
+For the purposes of this notice, WE refers to the entity of Flagrow and its proprietors, FORUM refers to a Flarum website that has installed Flagrow Bazaar and YOU refers to its owner, APPLICATION refers to Flagrow Bazaar, WEBSITE refers to flagrow.io. 
```

Comment:
If we are trying to create a privacy policy that can be held legally accountable and indemnify us, I think we need to be clear in our terms. Especially since general terms such as 'you' and 'the website' can have ambiguous interpretations, and at the point where we're defining those terms we might as well go whole hog.